scraper/src/youtube2zim/youtube.py

```diff
@@ -335,6 +459,25 @@ def extract_playlists_details_from(collection_type, youtube_id):
 
         # retrieve list of playlists for that channel
         playlist_ids = [p["id"] for p in get_channel_playlists_json(main_channel_id)]
+        
+        # Retrieve the shorts,long videos and lives playlist ID
+        user_long_uploads_playlist_id = get_user_long_uploads_playlist_id(main_channel_id)
```

Comment:
As mentioned above, `user_long_uploads_playlist_id = get_playlist_json("UULF" + main_channel_id[2:]")` can do this trick (and so on for other playlists) if you slightly modify `get_playlist_json` to return `None` when playlist is not found